src/test/java/io/pravega/connectors/flink/FlinkPravegaTableSinkTest.java

```diff
@@ -74,20 +76,23 @@ public void testBuilder() {
                 .withRoutingKeyField(TUPLE1.getFieldNames()[0]);
         FlinkPravegaWriter<Row> writer = builder.createSinkFunction(config);
         assertNotNull(writer);
+        FlinkPravegaOutputFormat<Row> outputFormat = builder.createOutputFormat(config);
+        assertNotNull(outputFormat);
```

Comment:
Added `outputFormat ` since we need additional parameter as an argument to the `FlinkPravegaTableSink` constructor. Added separate unit test to assert `FlinkPravegaOutputFormat`

Reply:
Please move lines 91 and 92 to be after 95, as a separate paragraph, and then add some assertion about the expected state of the `outputFormat`

Reply:
(done)